src/components/app-bar/BurgerMenu.svelte

```diff
@@ -0,0 +1,82 @@
+<script lang="ts">
+	import {
+		activeAccount,
+		activeAccountBounties,
+		walletConnect as wcConnection
+	} from '../../stores';
+	import { onMount } from 'svelte';
+
+	let open = false;
+	let container: HTMLDivElement | null = null;
+
+	function toggleBurgerMenu() {
+		open = !open;
+	}
+
+	$: {
+		if (open) {
+			window.addEventListener('click', closeMenuOnOutsideClick);
+		} else {
+			window.removeEventListener('click', closeMenuOnOutsideClick);
+		}
+	}
+
+	async function logOut() {
+		activeAccount.set(undefined);
+		sessionStorage.clear();
+		activeAccountBounties.set([]);
+		if ($wcConnection) {
+			await $wcConnection.disconnect();
+		}
+		closeBurgerMenu();
+	}
+
+	function closeBurgerMenu() {
+		open = false;
+	}
+
+	function closeMenuOnOutsideClick(event: MouseEvent) {
+		if (!container?.contains(event.target as Node)) {
+			closeBurgerMenu();
+		}
+	}
+
+	onMount(() => {
+		return () => {
+			window.removeEventListener('click', closeMenuOnOutsideClick);
+		};
+	});
+</script>
+
+<div bind:this={container} class="relative">
+	<!-- Burger Menu Icon -->
+	<button on:click={toggleBurgerMenu}>
```

Comment:
```suggestion
	<button on:click={() => { open = !open }}>
```
Could this become simpler now?